examples/xlnet/README.md

```diff
@@ -301,11 +272,11 @@ module in other tasks.
 #### Use your own dataset by writing a custom data processor
 
 It is easy to adapt the code to fine-tune XLNet on your custom dataset. To do this, you will need to write a custom
-data processor inheriting `xlnet.data.DataProcessor`. For concrete examples, please refer to the built-in processors
-under `xlnet/data/classification.py` and `xlnet/data/glue.py`.
+data processor inheriting `utils.processor.DataProcessor`. For concrete examples, please refer to the built-in processors
+under `utils/classification.py` and `utils/glue.py`.
```

Comment:
There are lots of files under `utils/`. Can we merge them? E.g., all data relevant stuff (data utils, processors, etc) can go into a single file. Make the file structure look as simple as possible

Reply:
BTW, what're the Processors for? What's their relation with Texar data modules?

Reply:
Maybe we can have a different folder for data-related files? Processors here are just data loader for different datasets in GLUE. After we use the processor loads the dataset (.tsv file). We will use texar data module to convert those files into pickled files.

Reply:
I see. At least all processors should be in a single file (now they are scattered in `processor`, `classification`, and `glue`, which looks complicated.).

Reply:
I'm actually in favor of separating data loaders by their tasks. We can group them together as a package.

Reply:
It's also good to separate by tasks, instead of `classification.py` which is confusing. The file name should be the task name.